src/main/java/org/folio/dew/batch/bulkedit/jobs/BulkEditItemProcessor.java

```diff
@@ -113,11 +127,12 @@ private String effectiveCallNumberComponentsToString(EffectiveCallNumberComponen
     if (isEmpty(components)) {
       return EMPTY;
     }
-    return String.join(ARRAY_DELIMITER,
-      isEmpty(components.getCallNumber()) ? EMPTY : escaper.escape(components.getCallNumber()),
-      isEmpty(components.getPrefix()) ? EMPTY : escaper.escape(components.getPrefix()),
-      isEmpty(components.getSuffix()) ? EMPTY : escaper.escape(components.getSuffix()),
-      escaper.escape(itemReferenceService.getCallNumberTypeNameById(components.getTypeId(), args)));
+    List<String> entries = new ArrayList<>();
+    ofEmptyString(components.getCallNumber()).ifPresent(e -> entries.add(escaper.escape(e)));
```

Comment:
same as in bulk-ops https://github.com/folio-org/stripes-util/blob/master/lib/effectiveCallNumber.js#L27